packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -867,11 +1000,20 @@ class SelectionOverlay {
 
   /// {@template flutter.widgets.SelectionOverlay.hideMagnifier}
   /// Hide the current magnifier, optionally immediately showing
-  /// the toolbar.
+  /// the toolbar provided by `contextMenuBuilder`.
   ///
   /// This does nothing if there is no magnifier.
   /// {@endtemplate}
-  void hideMagnifier({required bool shouldShowToolbar}) {
+  void hideMagnifier({
+    WidgetBuilder? contextMenuBuilder,
```

Comment:
nit: It seems a bit strange to hard-tie these 2 things together. If the developer wants to show the context menu immediately after hiding the magnifier, they should call `showToolbar`?

Reply:
Good call, I should have caught this in the loupe code review.  I'll change it.